### PR TITLE
samples: bluetooth: peripheral_ht: Health Thermometer sample

### PR DESCRIPTION
**Pull request by overheat**
Adding Health Thermometer Service sample. Refer to Health Thermometer
Profile Specification for detailed information about the Health
Thermometer Profile.

Signed-off-by: Aaron Tsui <aaron.tsui@outlook.com>

### CODE REVIEW

**Comment by overheat** [comment]
@aescolar any suggestion for the Shippable fail? I have no idea what it is.

File: samples/bluetooth/gatt/hts.c [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+		static u8_t htm[2];
+
+		temperature++;
+		if (temperature == 30U) {
+			temperature = 20U;
+		}
+
+		htm[0] = 0x0; /* uint8, temperature in celsius*/
+		htm[1] = temperature;
```
Comment (joerchan) [inline samples/bluetooth/gatt/hts.c]: The data format is supposed to be FLOAT (IEEE-11073 32-bit FLOAT).
Unfortunately there is no conversion helper function available that I know about.
So let's just do this:
```
		static u8_t htm[1 + 4];
		htm[0] = 0x0; /* uint8, temp in celsius*/
		/* (IEEE-11073 32-bit FLOAT). */
		htm[1] = temp; /* mantissa = temperature*/
		htm[2] = htm[3] = htm[4] = 0; /* sign = 0, exponent= 0 */
```
Then it seems to work.

**Review by joerchan — CHANGES_REQUESTED** [review]

**Comment by joerchan** [comment]
@overheat It appears to be a stability issue, the build has been re-triggered. Fingers crossed.

**Comment by aescolar** [comment]
@overheat 
About shippable: There is a few Shippable tests which are unstable and fail sometimes (which I understand are in the process of being fixed )
Some of us can re-trigger it independently when it fails, but you can also re-trigger all tests by closing+opening the PR, or pushing to the PR branch. 

Another thing is that, for some reason, it seems the other checks (documentation, gitlint, etc.) were hung. AFAIK, only @nashif can see why or re-trigger them independently.

File: samples/bluetooth/peripheral_ht/README.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
+.. _peripheral_ht:
+
+Bluetooth: Peripheral HT
+########################
+
+Overview
+********
+
+Similar to the :ref:`Peripheral <ble_peripheral>` sample, except that this
```
Comment (dbkinder) [inline samples/bluetooth/peripheral_ht/README.rst]: I'd prefer this to be a complete sentence, but I notice many of the Bluetooth samples are written with a similar sentence fragment, so I won't pick on yours.

**Review by dbkinder — APPROVED** [review]

File: samples/bluetooth/gatt/hts.c (hunk 1 above)
Comment (overheat) [reply samples/bluetooth/gatt/hts.c]: Done. Thanks.

**Review by joerchan — APPROVED** [review]

**Comment by joerchan** [comment]
@jhedberg Can we merge this one?

**Review by jhedberg — APPROVED** [review]

**Pull request merged**